src/apps/watchfaces/OswAppWatchface.cpp

```diff
@@ -55,7 +55,11 @@ void OswAppWatchface::drawStepHistory(OswUI* ui, uint8_t x, uint8_t y, uint8_t w
     }
 }
 #endif
-
+void OswAppWatchface::settingDisplayMainWatchface(String watchFaceIndex){
```

Comment:
As there is noone else using this -> could you merge that into the double-click handling of `handleButtonDefaults()`?